pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -109,12 +115,24 @@ func NewStaticPolicyOptions(policyOptions map[string]string) (StaticPolicyOption
 				return opts, fmt.Errorf("bad value for option %q: %w", name, err)
 			}
 			opts.AlignBySocket = optValue
+		case SpreadPCPUsPreferredOption:
+			optValue, err := strconv.ParseBool(value)
+			if err != nil {
+				return opts, fmt.Errorf("bad value for option %q: %w", name, err)
+			}
+			opts.SpreadPhysicalCPUsPreferred = optValue
+
 		default:
 			// this should never be reached, we already detect unknown options,
 			// but we keep it as further safety.
 			return opts, fmt.Errorf("unsupported cpumanager option: %q (%s)", name, value)
 		}
 	}
+
+	if opts.FullPhysicalCPUsOnly && opts.SpreadPhysicalCPUsPreferred {
+		return opts, fmt.Errorf("static policy options %s and %s can not be used at the same time", FullPCPUsOnlyOption, SpreadPCPUsPreferredOption)
+	}
```

Comment:
Do we also need a check against `DistributeCPUsAcrossNUMA`?  I haven't thought deeply about it myself, but on the surface it seems like the two policies would conflict.

Reply:
They do not work together at this moment. In future, I think we could make them work in theory and I left a TODO there

Reply:
I think we can leave as TODO for alpha, but this totally will be a beta blocker.